lib/html_parser/index.js

```diff
@@ -20,27 +20,32 @@ var tagValidator = require('./tag_validator');
  * @cb err, string
  * @pattern facade
  */
-function parse(htmlInput, prefix, postfix, cb) {
+function parse(htmlInput, prefix, postfix, sanitizerLogger, messageId, cb) {
   async.waterfall([
     // remove invalid pcdata, ie) <![if mso ...]>
     function(cb) {
+      sanitizerLogger && sanitizerLogger.info(`MessageId: ${messageId} HTML Parser: Remove invalid pcdata`);
```

Comment:
should these maybe be `debug` level logs? do we want then on permanently?

Reply:
Reverted back to info after talking to Ammar about this